data/human/human missions.txt

```diff
@@ -6890,3 +6890,109 @@ mission "Hogshead's Dilemma II"
+	description "Bring Caesar, a dog you found lost on <source>, to their owner on <destination>."
+	to offer
+		random < 15
+	source "Calda"
+	destination "Hestia"
+	on offer
+		conversation
+			`Your walk from the <ship> takes you outside <planet>'s bustling spaceport and into the overtly welcoming and carefully inoffensive tourist-saturated streets of the spaceport town. Every path seems purpose-built to whisk you further away from your ship and deeper into the unnatural radiance of this Paradise World. The hand-crafted beauty is inescapable, a never-ending sequence of breathtaking views.`
+			`	As you walk you eventually come across an immaculate and perfectly manicured garden park. Normally there is an exorbitant fee to enter, but due to some promotion today you can enter freely.`
+			choice
+				`	(Go to the park.)`
+					goto park
+				`	(Head back to your ship.)`
+
+			`	It was a splendid walk, but you decide it's time to head back. You have more important things to do than wander around an expensive terraformed garden.`
+				decline
+
+			label park
+			`	The park has been meticulously sculpted so that no matter where you stand it appears as though you are in the center of a lush naturalist painting. Colorful, comforting flora bleed away from you at all angles, and adorable yet inaccessible fauna seem to hug the very periphery of your every step.`
+			`	The air is clear and crisp, and even the crunch and thud of the ground beneath your feet seem somehow delicious and satisfying. Finally, the sun begins and set and it is time to head back.`
+			`	As you approach the exit to the park a small dog pads over to you and looks at you with morunful eyes. The dog seems to be alone with no owner anywhere in sight.`
+			`	"Come here," you call to the poodle-hybrid, who obediently does as you command. You see that it is a male and check its collar:`
+			``
+			`	"Caesar Augustus - Property of the Mitchells"`
+			`	3 Mitchell Drive`
+			`	Avalonia District, Hestia`
+			``
+			`	"That's odd," you say to Caesar. "How did you get lost on the wrong planet, huh boy?"`
+			`	You pet Caesar, who seems glad of the attention, and decide what to do.`
+			choice
+				`	(Take Caesar to his owners on Hestia.)`
+					goto takehim
+				`	(Leave Caesar here.)`
+
+			`	Perhaps his owners are on vacation and will be back looking for him any minute. Although it's getting dark and no one seems to be around, you decide the best course of action to leave Caesar at the park.`
+			`	As you walk off you hear Caesar whining, and as you look back you notice him slink off into the park. But you've made your decision; hopefully everything will work out for the best.`
+				decline
+
+			label takehim
+			`	"Well, come on boy. Let's go." Caesar eagerly follows you back to your ship. He seems greatly relieved to be with you, and not left all alone at the park.`
+			`	Once back on your ship, you make sure he's fed, watered, and settled in for his interstellar journey.`
```

Comment:
```suggestion
			`	Once back on the <ship>, you make sure he's fed, watered, and settled in for his interstellar journey.`
```

Reply:
Actually your suggestion was the old version. But I then suggested to replace it by the current one.

Reply:
I like `your`

Reply:
Ah, okay then.